### PR TITLE
Update base64 to 0.21

### PR DESCRIPTION
**Pull request by sameo**
And fix a resolver issue.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -6,6 +6,7 @@ members = [
     "bin/grpc-as",
     "bin/rvps-client",
 ]
+resolver = "2"
```
Comment (mkulke) [inline Cargo.toml:9]: i'm curious, why is it necessary? I read it's implied by setting `edition = 2021`, which might be less obscure. but fwiw I don't get any compilation errors when removing that line.

**Review by fitzthum — APPROVED** [review]
Thanks. I was too lazy to do this when I added the snp verifier.

**Review by Xynnn007 — APPROVED** [review]
Thanks for this!

**Review by jialez0 — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (sameo) [reply Cargo.toml:10]: I see this linter warning with 1.72 when running a basic `cargo build`. Some of our dependencies are explicitly setting `edition = 2021` which implies `resolver = 2` which is different from the the virtual workspace (we do use a virtual workspace) default resolver.

I feel like I'm rephrasing the warning itself, not sure this is giving a better explanation...

**Review by mkulke — APPROVED** [review]

**Pull request merged**